src/installer/pkg/sfx/Microsoft.NETCore.App/ReadyToRun.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
I don’t like these Imports not being in the project files directly. Can we move these imports back to the .sfxproj files?

Reply:
No problem, fixed in 2nd commit.